compliance/notify.py

```diff
@@ -552,6 +552,14 @@ def _generate_issues(self, accred, results):
         if not results:
             return issues
         for check_path, result, message in results:
+            # If the 'checks' configuration element exists
+            # within an accreditation, only create issues
+            # for the set of checks therein.
+            if (isinstance(self._config[accred], dict)
+                    and 'checks' in self._config[accred]):
+                checks = self._config[accred]['checks'] or []
+                if checks and check_path not in checks:
+                    continue
```

Comment:
I don't think that the accreditation configuration for a GH notifier can be anything but a dict so I don't know that `isinstance(self._config[accred], dict)` is necessary?  Also I don't know that all of `if checks and check_path not in checks:` is needed either.  IMO if a "checks" config is added it should be honored as is.  So this all could probably be collapsed down to just:

```suggestion
            if ('checks' in self._config[accred].keys() 
                    and check_path not in self._config[accred]['checks']):
                continue
```

Reply:
OK, done. Thanks!